tests/cxx/built_ins_template.cc

```diff
@@ -0,0 +1,20 @@
+//===--- built_ins_template.cc - test input file for iwyu -----------------===//
```

Comment:
"builtins" is usually written as one word, so maybe `builtin_templates.cc` instead?

Reply:
I was following the convention of the other two tests: `built_ins_no_includes.cc` and `built_ins_new_included.cc`. Personally, I prefer *builtins*, should I make them all consistent?

Reply:
Ah, I see. Let's leave it as-is for now, and we can follow up with a global rename later.